code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -162,6 +162,13 @@
 	list_reagents = list(/datum/reagent/medicine/kelotane = 15)
 	rename_with_volume = TRUE
 
+/obj/item/reagent_containers/pill/tricordrazine_first_aid
+	name = "Washes Pain Away pill"
+	desc = "A simple pill that tends all small damages. You might need to read an drug instruction in the pill bottle..."
+	icon_state = "pill_happy"
+	list_reagents = list(/datum/reagent/medicine/tricordrazine = 36)
```

Comment:
Why do they have 36u each when 30u is the OD limit for tricordrazine?

Reply:
because its OD is now 50u and tricordrazine is pretty weak now.

Reply:
Oh right, that other PR just got merged.